### PR TITLE
Fix for Resource Listing - BoxResource

### PR DESCRIPTION
**Pull request by ImVis10**
Resolves a part of https://github.com/didclab/onedatashare/issues/424

- [x] BoxResource

### CODE REVIEW

**Comment by ImVis10** [comment]
image

Last Modified time and folder size getting reflected correctly

File: src/main/java/org/onedatashare/server/service/CredentialService.java
Comment (jgoldverg) [inline src/main/java/org/onedatashare/server/service/CredentialService.java:1]: Undo this change as it was only for local testing.

File: src/main/resources/application.properties
Comment (jgoldverg) [inline src/main/resources/application.properties:1]: undo this change as it was only for local testing

File: pom.xml [hunk 1]
```diff
@@ -313,10 +314,6 @@
             <version>3.1.0</version>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-actuator</artifactId>
```
Comment (jgoldverg) [inline pom.xml:318]: why remove actuator?

File: pom.xml [hunk 2]
```diff
@@ -14,15 +14,15 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.0.3.RELEASE</version>
+        <version>2.7.9</version>
```
Comment (jgoldverg) [inline pom.xml]: can we try bumping to 2.7.10 as thats the latest one.

File: pom.xml (hunk 1 above)
Comment (ImVis10) [reply pom.xml:318]: actuator is present twice, which is why I decided to do this.

File: pom.xml (hunk 2 above)
Comment (ImVis10) [reply pom.xml]: updated

**Review by jgoldverg — APPROVED** [review]
Everything checks out :)
